### DISCUSSION TITLE
Is there a manual?

### DISCUSSION THREAD

**Discussion by Schlaunz** [apohl79/audiogridder#905] [Help]
Hello Everyone,

I hope you are all doing well.

Today  I found the Audiogridder Sotftware.

Really interesting Idea.

I downloaded and tested right away.

It's working but not without issues.

I am completely new to this and before asking a bunch of silly questions and waisting people's time; is there something like a manual?

Thanks a lot

Stay safe

**Comment by CaBuffalo** [comment]
Hi, there:

Sorry we don't have a manual available. Please refer to our website or
forum if you have any questions.

Thanks.

On Wed, Feb 2, 2022 at 2:15 PM Schlaunz ***@***.***> wrote:

> Hello Everyone,
>
> I hope you are all doing well.
>
> Today I found the Audiogridder Sotftware.
>
> Really interesting Idea.
>
> I downloaded and tested right away.
>
> It's working but not without issues.
>
> I am completely new to this and before asking a bunch of silly questions
> and waisting people's time; is there something like a manual?
>
> Thanks a lot
>
> Stay safe
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/905>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GWNG6KOFOAOENPDODUZGUHFANCNFSM5NNI54XA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Schlaunz** [comment]
Hello Cabuffalo,

Thanks

Best Regards
